src/ui/components/DashKit/plugins/mixins.scss

```diff
@@ -0,0 +1,14 @@
+@mixin with-fade-effect() {
+    content: '';
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    width: 44px;
+    right: 0;
+    background: linear-gradient(
+        to right,
+        transparent,
+        color-mix(in sRGB, var(--g-color-base-generic-ultralight) 95%, transparent),
```

Comment:
`color-mix` arrived in support in 2023... and not supported in WebView do we really has no other options?